### PR TITLE
remove detected code smells #34

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

File: src/main/java/com/github/cameltooling/model/BaseOptionModel.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ */
+package com.github.cameltooling.model;
+
+/**
+ * @author lheinema
+ */
+public class BaseOptionModel {
+
+	private String name;
+    private String kind;
```
Comment (apupier) [inline src/main/java/com/github/cameltooling/model/BaseOptionModel.java]: space instead of tabs used

**Review by apupier — COMMENTED** [review]
I'm not sure that we should fix them now.
The idea was to stay as close as possible to IntelliJ plugin code so that factorization would be easier.
Do you think that it has an higher priority to have cleaner code than staying in sync with IntelliJ plugin code?

i'm wondering if some of the code is not generated code, in this case it should be marked as generated and duplication would be "normal".

**Comment by lhein** [comment]
I think it would make more sense to move the intelliJ plugin to use this common library instead. @davsclaus @bennetelli @fharms any ideas on that?

**Comment by apupier** [comment]
archiving this repo

**Pull request closed without merging**